arches/app/media/js/views/graph/graph-designer/node-form.js

```diff
@@ -33,6 +33,29 @@ define([
                 return self.graphModel.get('isresource') && node && node.istopnode;
             });
             this.restrictedNodegroups = options.restrictedNodegroups;
+            this.appliedFunctions = options.appliedFunctions;
+
+            this.isFuncNode = function() {
+                var node = self.node();
+                var appFuncDesc = this.appliedFunctions()[0]['config']['description']['nodegroup_id'];
```

Comment:
this section has the same problems as described above re: isFuncNode in card.js, ie keying on properties that may not be present and assuming the function of interest is at index 0 of the array.